docs/contributing/code-style.md

```diff
@@ -198,8 +198,7 @@ number without any explicit conversion.
 ### JavaScript `const` and `let`
 
 Always declare JavaScript variables using `const` or `let` rather than
-`var`, except in the Casper tests (since Casper does not support
-`const` and `let`).
+`var`.
```

Comment:
@andersk is this something we can move to linters now?

Reply:
We’ve been linting this since #13357, with a Casper-specific exemption that this PR has correctly removed.

Reply:
Great, thanks for the confirmation!